ci/docker/image-builder/.dockerignore

```diff
@@ -0,0 +1,6 @@
+# Include only app sources for build
```

Comment:
I need explanation for what that file exists

Reply:
https://docs.docker.com/engine/reference/builder/#dockerignore-file
This is a standard config to reduce the amount of copied files while building an image.

Reply:
Thank you. I will take it into account